html/modules/custom/reliefweb_import/src/Command/ReliefwebImportCommand.php

```diff
@@ -597,6 +597,34 @@ protected function validateCity($data) {
     return $field_city;
   }
 
+  /**
+   * Validate job closing date field.
+   *
+   * @param string $data
+   *   Raw data from XML.
+   */
+  protected function validateJobClosingDate($data) {
+    // Clean the field.
+    $field_job_closing_date = mb_substr($data, 0, 10);
+
+    // Ensure the field size is reasonable.
+    $length = mb_strlen($field_job_closing_date);
+    if ($length !== 0 && $length !== 10) {
+      throw new ReliefwebImportExceptionSoftViolation(strtr('Invalid data for field_job_closing_date, @length characters found, format has to be yyyy-mm-dd', [
+        '@length' => $length,
+      ]));
+    }
+
+    // Make sure field can be converted to a date.
+    if ($length === 10 && !strtotime($field_job_closing_date)) {
+      throw new ReliefwebImportExceptionSoftViolation(strtr('Invalid data for field_job_closing_date, @data has to in format yyyy-mm-dd', [
```

Comment:
> @data has to in format yyyy-mm-dd

Typo.

Reply:
It's meant to be `data`, since it's the user input